src/main/java/frc/trigon/robot/RobotContainer.java

```diff
@@ -48,21 +56,39 @@ private void initComponents() {
 
     private void initCommands() {
         swerveCmd = new FieldRelativeSupplierDrive(
-                () -> -controller.getLeftY(),
-                () -> controller.getLeftX(),
+                () -> controller.getLeftY(),
+                () -> -controller.getLeftX(),
                 () -> -controller.getRightX()
         );
         collectCommand = new CollectCommand();
 
         countBallsCommand = new CountBallsCommand();
         shotsDetectorCommand = new ShotsDetectorCommand();
+
+        turnToHubCmd = new TurnToTargetCommand(limelight, 0);
+
+        primeShooterCmd = Shooter.getInstance().getPrimeShooterCommandWithDefault(
+                () -> limelight.hasTarget() ?
+                      ShootingCalculations.getShootingVelocityFromDistance(limelight.getDistanceFromHub()) :
+                      0
+        );
+        pitchCmd = Pitcher.getInstance().getPitchingCommandWithDefault(
+                () -> limelight.hasTarget() ?
+                      ShootingCalculations.getShootingAngleFromDistance(limelight.getDistanceFromHub()) :
+                      0
+        );
     }
 
     private void bindCommands() {
         Swerve.getInstance().setDefaultCommand(swerveCmd);
+        Pitcher.getInstance().setDefaultCommand(pitchCmd);
+        Shooter.getInstance().setDefaultCommand(primeShooterCmd);
+
         controller.getABtn().whileHeld(collectCommand);
         controller.getYBtn().whenPressed(Swerve.getInstance()::zeroHeading);
-        controller.getBBtn().whileHeld(Loader.getInstance().getLoadCommand());
+        controller.getBBtn().whileHeld(
+                Loader.getInstance().getLoadCommand().alongWith(BackspinReducer.getInstance().getReducerCommand()));
```

Comment:
If you are running the shooter the whole time, then there is no reason not to always run the backspinReducer